image_source_stream.go

```diff
@@ -23,7 +39,21 @@ func streamSource(ctx context.Context, once func(), is ImageSource, stream Strea
 		frame, release, err := is.Next(ctx)
 		if err != nil {
 			Logger.Debugw("error getting frame", "error", err)
+			if prevErr == nil {
```

Comment:
I thought you were going to do this in rdk, not here?

Reply:
so, @maximpertsov and I followed the breadcrumbs on where we were managing the receipt frames and they actually just led right back to here. On reflection, I also think this is general enough behavior that we would want it to apply to all streams (though perhaps we want to provide an ability to tune the backoff from the outside?)

Reply:
that's okay. if it lives here, this should be an option to use or not and yes ability to tune it.